examples/connext_dds/CMakeLists.txt

```diff
@@ -10,9 +10,12 @@ set(CMAKE_MODULE_PATH
     "${CMAKE_CURRENT_SOURCE_DIR}/../../resources/cmake"
 )
 
-find_package(RTIConnextDDS
+set(REQUIRED_VERSION
```

Comment:
Instead of doing this, I think it is a better idea to set directly the version in each `if` statement. The reason is that this `REQUIRED_VERSION` will change depending on the new features added to the product so, it will be easier to scale the number of examples in the future.